aptos-move/framework/aptos-framework/sources/delegation_pool.move

```diff
@@ -221,6 +243,48 @@ module aptos_framework::delegation_pool {
         distribute_commission_events: EventHandle<DistributeCommissionEvent>,
     }
 
+    struct VotingRecordKey has copy, drop, store {
+        voter: address,
+        proposal_id: u64,
+    }
+
+    struct VotingPowerDelegation has copy, drop, store {
```

Comment:
Can you add documentation in front of each struct here to explain what data it holds and where it's stored/used?